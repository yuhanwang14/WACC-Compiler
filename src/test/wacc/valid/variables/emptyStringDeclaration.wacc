# simple empty string variable declaration

# Output:

# Program:

begin
  string s = ""
end
