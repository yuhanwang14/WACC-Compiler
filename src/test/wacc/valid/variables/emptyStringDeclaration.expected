 1
 2
 3
 4
 5
 6
 7
 8
 9
10
# simple empty string variable declaration

# Output:

# Program:

begin
  string s = ""
end
 